products/silverstripe.md

```diff
@@ -25,40 +25,47 @@ identifiers:
 -   purl: pkg:github/silverstripe/cms
 
 releases:
+-   releaseCycle: "5.1"
+    releaseDate: 2023-10-16
+    support: 2024-04-01
+    eol: 2027-04-01
```

Comment:
I see April 2026 and April 2027 for support and eol dates. But this is for 5.x, not 5.1. I think we should just set true / false like for 5.0. WDYT @BiNZGi  ?

Reply:
This was from https://www.silverstripe.org/software/roadmap/ I just saw that they inserted a new row for 5.1 with new dates, I update the MR...